packages/node_modules/@ciscospark/widget-recents/src/components/spaces-list/index.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react';
```

Comment:
Are we just leaving these components out of public components until they are needed to be shared? I'm good with it, just confirming.